### ISSUE TITLE
U of CRUD Friends

### ISSUE THREAD

**Issue by thedigitalmenagerie** [thedigitalmenagerie/hr-gratitude-capstone#23]
## Feature Summary

As a user, I should be able to update other users' accessibility to my content. 

## Acceptance Criteria

- [ ] All users being able to update friends' access to their content.

## Technical Requirements

<!-- EXAMPLE:
- [ ] User update with checkbox
